technology-considerations/droid_vs_siegfried.md

```diff
@@ -0,0 +1,22 @@
+## Droid vs Siegfried
```

Comment:
Could you move this to the [Architecture Decisions Records](https://github.com/nationalarchives/tdr-dev-documentation/tree/master/architecture-decision-records) folder, since we've now made the decision to go with DROID?

See the other ADRs for the structure - just add a decision date and a sentence with the option we're going with.

Reply:
I've moved it into the ARDs folder and updated the structure to match the others.